plugins/stylelint-config-atlas/index.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+module.exports = {
+	plugins: ['stylelint-scss', 'stylelint-order'],
```

Comment:
One missed suggested dependency which is the standard config (I think other plugins/configs usually assume it's being added). It catches things like descending specificity (which explodes that test file haha) and my personal favorite of ensuring `::after` vs `:after` since single colon is technically deprecated (the fix option will resolve current instances)

```suggestion
        extends: ['stylelint-config-standard'],
	plugins: ['stylelint-scss', 'stylelint-order'],
```

Reply:
updated!